src/lib/Repository/Values/Content/UiVersionInfo.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\HybridPlatformUi\Repository\Values\Content;
+
+use eZ\Publish\Core\Repository\Values\Content\VersionInfo;
+use eZ\Publish\API\Repository\Values\Content\VersionInfo as APIVersionInfo;
+
+/**
+ * Extends original value object in order to provide additional fields.
+ * Takes a standard VersionInfo instance and retrieves properties from it in addition to the provided properties.
```

Comment:
Damn, something I've forgotten since the beginning: we need to document the properties we add to the value objects:

```
/**
 * @property-read \eZ\Publish\API\Repository\Values\User\User $author
 */
```

Applies to all other objects of course :p